kolibri/plugins/coach/assets/src/views/create-exam-page/index.vue

```diff
@@ -225,6 +257,11 @@
         }
         return '';
       },
+      generateRandomSeed() {
```

Comment:
I think it would depend on the situation. To me, replacing this with `lodash/random` seems like a borderline case.